guides/common/modules/proc_deploying-a-custom-ssl-certificate-to-hosts.adoc

```diff
@@ -27,12 +27,6 @@ ifndef::satellite,orcharhino[]
 ----
 # {client-package-install-el8} http://_{common-example-com}_/pub/katello-ca-consumer-latest.noarch.rpm
 ----
-** On {EL} 7:
-+
-[options="nowrap" subs="+quotes,attributes"]
-----
-# {client-package-install-el7} http://_{common-example-com}_/pub/katello-ca-consumer-latest.noarch.rpm
-----
```

Comment:
Can we keep this for Satellite when it's on hosts?

Reply:
Same as below. I only remove EL7 for Non-Satellite/orcharhino. See line 9/15.